src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -51,6 +55,10 @@ import { calculateStatus$ } from './status';
 import { registerCoreObjectTypes } from './object_types';
 import { getSavedObjectsDeprecationsProvider } from './deprecations';
 
+export const ENCRYPTION_EXTENSION_ID = 'encryptedSavedObjects' as const;
+export const SECURITY_EXTENSION_ID = 'security' as const;
+export const SPACES_EXTENSION_ID = 'spaces' as const;
```

Comment:
These SO extension IDs are the same as the plugins that provide these extensions. This is purely for backwards compatibility for consumers that currently create a scoped SOC and disable one of the SOC wrappers.